lib/improver/utilities/cube_metadata.py

```diff
@@ -40,29 +40,40 @@
 from improver.utilities.cube_manipulation import compare_coords
 
 
+GRID_TYPE = 'standard'
+STAGE_VERSION = '1.3.0'
+
+# Define current StaGE grid metadata
+MOSG_GRID_DEFINITION = {
+    'uk_ens': {'mosg__grid_type': GRID_TYPE,
+               'mosg__model_configuration': 'uk_ens',
+               'mosg__grid_domain': 'uk_extended',
+               'mosg__grid_version': STAGE_VERSION},
+    'gl_ens': {'mosg__grid_type': GRID_TYPE,
+               'mosg__model_configuration': 'gl_ens',
+               'mosg__grid_domain': 'global',
+               'mosg__grid_version': STAGE_VERSION},
+    'uk_det': {'mosg__grid_type': GRID_TYPE,
+               'mosg__model_configuration': 'uk_det',
+               'mosg__grid_domain': 'uk_extended',
+               'mosg__grid_version': STAGE_VERSION},
+    'gl_det': {'mosg__grid_type': GRID_TYPE,
+               'mosg__model_configuration': 'gl_det',
+               'mosg__grid_domain': 'global',
+               'mosg__grid_version': STAGE_VERSION}
+}
+
+
 # Define correct v1.2.0 meta-data for v1.1.0 data.
-GRID_ID_LOOKUP = {'enukx_standard_v1': {'mosg__grid_type': 'standard',
-                                        'mosg__model_configuration': 'uk_ens',
-                                        'mosg__grid_domain': 'uk_extended',
-                                        'mosg__grid_version': '1.1.0'},
-                  'engl_standard_v1': {'mosg__grid_type': 'standard',
-                                       'mosg__model_configuration': 'gl_ens',
-                                       'mosg__grid_domain': 'global',
-                                       'mosg__grid_version': '1.1.0'},
-                  'ukvx_standard_v1': {'mosg__grid_type': 'standard',
-                                       'mosg__model_configuration': 'uk_det',
-                                       'mosg__grid_domain': 'uk_extended',
-                                       'mosg__grid_version': '1.1.0'},
-                  'glm_standard_v1': {'mosg__grid_type': 'standard',
-                                      'mosg__model_configuration': 'gl_det',
-                                      'mosg__grid_domain': 'global',
-                                      'mosg__grid_version': '1.1.0'},
-                  }
+GRID_ID_LOOKUP = {'enukx_standard_v1': 'uk_ens',
+                  'engl_standard_v1': 'gl_ens',
+                  'ukvx_standard_v1': 'uk_det',
+                  'glm_standard_v1': 'gl_det'}
 
 
 def stage_v110_to_v120(cube):
     """Translates meta-data relating to the grid_id attribute from StaGE
-    version 1.1.0 to StaGE version 1.2.0.
+    version 1.1.0 to later StaGE versions.
```

Comment:
It would make sense in this case, to also update the name of this function, in order to avoid confusion later on.